sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/b2c/AADB2CProperties.java

```diff
@@ -68,6 +68,11 @@
 
     private Map<String, Object> authenticateAdditionalParameters;
 
+    /**
+     * User name attribute name,eg "name"
+     */
+    private String UserNameAttributeName;
```

Comment:
Please update related README and CHANGELOG.

Reply:
The variable name should started by lower case.

Reply:
all right